### DISCUSSION TITLE
NRF52840-DK to donate

### DISCUSSION THREAD

**Discussion by rsporsche** [ffenix113/zigbee_home#39] [General]
I have a NRF52840-DK (developer kit) that I would be happy to donate if it's of use. 

I bought it hoping to create a battery powered zigbee relay before discovering the 'fingerbot'.

**Comment by lboue** [comment]
> I have a NRF52840-DK (developer kit) that I would be happy to donate if it's of use.
> 
> I bought it hoping to create a battery powered zigbee relay before discovering the 'fingerbot'.

@rsporsche
I'm interested in this NRF52840-DK (developer kit). I would use it to test Matter over Thread integration with HomeAssistant/ESPHome.

**Reply by rsporsche** [reply]
Hi Iboue,

I sent it to ffenix since this is his project.

You would have been next in line though!

**Comment by ffenix113** [comment]
Hello @rsporsche ,

Could you please clarify if you would like to donate the board to this project, or to anyone for whom it could be useful?

Anyhow - I am thankful about your contribution, it is very generous of you.

**Reply by rsporsche** [reply]
Sorry, I realise I wasn't very specific.

My intention is to donate it to benefit this specific project since it's of interest to me.

If it's not needed by anyone working on this project I would be happy to donate it to someone else if it benefited another home automation project or someone who actively contributes to home automation project but I'm not interested in just giving it away to be used as a toy.

I'm based in Germany and am prepared to cover the cost of postage within reason. Also happy to send to a pickup location to maintain privacy.

**Reply by ffenix113** [reply]
The board would be definitely of use and helpful, and if you indeed would decide that you are able to donate the board to this project - it would be amazing.

Could you please contact me [here](https://airtable.com/appykiuiUYh56G6qD/shrDJxMEnnbmAZhIn)(including your email) and I will write you back.

Thank you.

**Reply by rsporsche** [reply]
Done

**Reply by ffenix113** [reply]
Board arrived and is in as-new condition. Thank you @rsporsche .
